release/sdk-config.yaml.tmpl

```diff
@@ -1,50 +1,36 @@
 version: __VERSION__
 commands:
-- name: compile
-  path: damlc/da-hs-damlc-app
-  args: ["compile"]
-  desc: "Compile a DAML file"
-- name: package
-  path: damlc/da-hs-damlc-app
-  args: ["package-new"]
-  desc: "Compile a DAML project into a DAR"
-- name: init
-  path: damlc/da-hs-damlc-app
-  args: ["init"]
-  desc: "Initialize a DAML project"
-- name: build
-  path: damlc/da-hs-damlc-app
-  args: ["build"]
-  desc: "Initialize, build and package a DAML project"
-- name: test
-  path: damlc/da-hs-damlc-app
-  args: ["test"]
-  desc: "Run the scenarios in the given DAML file and all dependencies"
-- name: ide
-  path: damlc/da-hs-damlc-app
-  args: ["lax", "ide"]
-  desc: "Run the DAML language server"
 - name: studio
   path: daml-helper/daml-helper
   desc: "Launch DAML studio"
   args: ["studio"]
+- name: new
+  path: daml-helper/daml-helper
+  desc: "Create a new project"
+  args: ["new"]
+- name: build
+  path: damlc/da-hs-damlc-app
+  args: ["build"]
+  desc: "Build project"
+- name: start
+  path: daml-helper/daml-helper
+  args: ["start"]
+  desc: "Run sandbox and navigator in project"
+- name: damlc
+  path: damlc/da-hs-damlc-app
+  desc: "Launch the DAML compiler"
```

Comment:
Surely the compiler is something you run? I guess launch to me implies an ongoing process.

Reply:
Agreed. Run